utils/ccloud_library.sh

```diff
@@ -69,50 +69,49 @@ function ccloud::validate_expect_installed() {
 
   return 0
 }
-function ccloud::validate_ccloud_cli_installed() {
-  if [[ $(type ccloud 2>&1) =~ "not found" ]]; then
-    echo "'ccloud' is not found. Install Confluent Cloud CLI (https://docs.confluent.io/ccloud-cli/current/install.html) and try again"
+function ccloud::validate_cli_installed() {
+  if [[ $(type confluent 2>&1) =~ "not found" ]]; then
+    echo "'confluent' is not found. Install the Confluent CLI (https://docs.confluent.io/confluent-cli/current/install.html) and try again."
     exit 1
   fi
 }
 
-function ccloud::validate_ccloud_cli_v2() {
-  ccloud::validate_ccloud_cli_installed || exit 1
+function ccloud::validate_cli_v2() {
+  ccloud::validate_cli_installed || exit 1
 
-  if [[ -z $(ccloud version 2>&1 | grep "Go") ]]; then
-    echo "This example requires the new Confluent Cloud CLI. Please update your version and try again."
+  if [[ -z $(confluent version 2>&1 | grep "Go") ]]; then
+    echo "This example requires the new Confluent CLI. Please update your version and try again."
     exit 1
   fi
 
   return 0
 }
 
-function ccloud::validate_logged_in_ccloud_cli() {
-  ccloud::validate_ccloud_cli_v2 || exit 1
+function ccloud::validate_logged_in_cli() {
+  ccloud::validate_cli_v2 || exit 1
 
-  if [[ "$(ccloud kafka cluster list 2>&1)" =~ "ccloud login" ]]; then
+  if [[ "$(confluent kafka cluster list 2>&1)" =~ "confluent login" ]]; then
     echo
     echo "ERROR: Not logged into Confluent Cloud."
-    echo "Log in with the command 'ccloud login --save' before running the example. The '--save' argument saves your Confluent Cloud user login credentials or refresh token (in the case of SSO) to the local netrc file."
+    echo "Log in with the command 'confluent login --save' before running the example. The '--save' argument saves your Confluent Cloud user login credentials or refresh token (in the case of SSO) to the local netrc file."
     exit 1
   fi
 
   return 0
 }
 
-function ccloud::get_version_ccloud_cli() {
-  ccloud version | grep "^Version:" | cut -d':' -f2 | cut -d'v' -f2
+function ccloud::get_version_cli() {
+  confluent version | grep "^Version:" | cut -d':' -f2 | cut -d'v' -f2
 }
 
-function ccloud::validate_version_ccloud_cli() {
+function ccloud::validate_version_cli() {
+  ccloud::validate_cli_installed || exit 1
 
-  ccloud::validate_ccloud_cli_installed || exit 1
+  CLI_VERSION=$(ccloud::get_version_cli)
 
-  CCLOUD_VER=$(ccloud::get_version_ccloud_cli)
-
-  if ccloud::version_gt $CCLOUD_MIN_VERSION $CCLOUD_VER; then
-    echo "ccloud version ${CCLOUD_MIN_VERSION} or greater is required.  Current reported version: ${CCLOUD_VER}"
-    echo 'To update run: ccloud update'
+  if ccloud::version_gt $CLI_MIN_VERSION $CLI_VERSION; then
+    echo "confluent version ${CLI_MIN_VERSION} or greater is required. Current version: ${CLI_VERSION}"
+    echo "To update run: confluent update"
```

Comment:
IIRC the recommendation is not to do `update`, per https://docs.confluent.io/confluent-cli/current/migrate.html